### PR TITLE
Add missing Profile Links of Speakers

### PR DESCRIPTION
**Pull request by Ploypiti**
#83 (GCI tasks)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -953,7 +953,8 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="_self" href="https://twitter.com/eugeneteo"><i class="icon social_twitter"></i></a>
 											<a target="_self" href="http://sg.linkedin.com/in/eugeneteo"><i class="icon social_linkedin"></i></a>
-										</div>
+                      <a target="_self" href="https://web.facebook.com/eugenete0"><i class="icon social_facebook"></i></a>
+                    </div>
```
Comment (harshit98) [inline index.html]: Lint these 2 lines properly.

**Comment by harshit98** [comment]
@Ploypiti Kindly lint all the lines in which you have done the changes.

**Comment by Ploypiti** [comment]
@harshit98 Can you please approve without squashing? Thx.

**Review by isuruAb — COMMENTED** [review]
please indent the code. and squash your commits.
follow this tutorial to learn about squashing.
https://www.youtube.com/watch?v=vp7-NeURmFg

**Comment by harshit98** [comment]
Try to squash commits @Ploypiti : )

File: index.html [hunk 2]
```diff
@@ -1100,7 +1101,7 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="_self" href="https://twitter.com/alyssaquek"><i class="icon social_twitter"></i></a>
 											<a target="_self" href="http://sg.linkedin.com/in/alyssaq"><i class="icon social_linkedin"></i></a>
-
+                      <a target="_self" href="https://web.facebook.com/alyssaq"><i class="icon social_facebook"></i></a>
```
Comment (harshit98) [inline index.html]: Indent this line.

File: index.html [hunk 3]
```diff
@@ -1161,7 +1162,8 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="_self" href="https://twitter.com/gravax"><i class="icon social_twitter"></i></a>
 											<a target="_self" href="http://ch.linkedin.com/in/gillesgravier"><i class="icon social_linkedin"></i></a>
-										</div>
+                      <a target="_self" href="https://web.facebook.com/ggravier"><i class="icon social_facebook"></i></a>
+                    </div>
```
Comment (harshit98) [inline index.html]: Indent these two lines.

File: index.html [hunk 4]
```diff
@@ -1353,7 +1355,7 @@ <h1>Speakers</h1>
 									<img class="background-image" alt="Ricky Setyawan" src="img/rickysetyawan.jpg">
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
-											
+                      <a target="_self" href="https://twitter.com/dbmsboy"><i class="icon social_twitter"></i></a>
```
Comment (harshit98) [inline index.html]: Indent this line.

File: index.html [hunk 5]
```diff
@@ -2295,7 +2297,8 @@ <h1>Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="_self" href="https://twitter.com/shwetank"><i class="icon social_twitter"></i></a>
-											<a target="_self" href="http://in.linkedin.com/pub/shwetank-dixit/2/a6/b43/en"><i class="icon social_linkedin icon-large"></i></a>
+                      <a href="https://www.shwetankdixit.com/"><i class="icon  icon_link_alt" aria-hidden="true"></i></a>
+                      <a target="_self" href="http://in.linkedin.com/pub/shwetank-dixit/2/a6/b43/en"><i class="icon social_linkedin icon-large"></i></a>
```
Comment (harshit98) [inline index.html]: Indent these two lines as well.

**Comment by Ploypiti** [comment]
I'm trying to pull the branch
```
C:\Users\P10YP1T1\2015.fossasia.org>git pull origin patch-1 -f
fatal: Couldn't find remote ref patch-1
```
_Deads_

**Review by harshit98 — APPROVED** [review]

**Pull request merged**